plugins/builddep.py

```diff
@@ -62,7 +63,7 @@ class BuildDepCommand(dnf.cli.Command):
 
     def __init__(self, cli):
         super(BuildDepCommand, self).__init__(cli)
-        self.rpm_ts = rpm.TransactionSet()
+        self._rpm_ts = dnf.rpm.transaction.initReadOnlyTransaction()
```

Comment:
it disables checking of digests (checksums). According to Florian it's harmless for SRPMs and gives speedup.